index.html

```diff
@@ -7,5 +7,200 @@
 </head>
 <body>
 
+<!-------------------------------------------------- Content header --------------------------------------------------->
+<header class="outer-container column">
+    <nav class="navbar row">
+        <ul class="row">
+            <li><a href="#work">Case Studies</a></li>
+            <li><a href="#services">Services</a></li>
+            <li><a href="#contact">Contact Us</a></li>
+        </ul>
+    </nav>
+    <div class="hero-section column">
+        <h1>Your website is an investment in the success of your business.</h1>
+        <p>We are here to help you succeed.</p>
+    </div>
+    <button type="button" class="cta-btn"><a href="#contact">Let's work together</a></button>
+    <div class="squiggle-line-stripe"><img src="./assets/squiggle_line_stripe.svg" alt="squiggle-line"></div>
+</header>
+
+<!----------------------------------------------- Content hero 2 section ---------------------------------------------->
+<div class="triangle-bottom-left"></div>
+<div class="hero2-section column">
+
+    <div></div>   <!--  div for big font "hello"  -->
+    <p>Rekupe is a digital agency in Los Angeles.</p>
+    <p>We use strategic UX, SEO optimization and beautiful visual designs to help your business grow</p>
+    <p>We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.</p>
+
+</div>
+
+<ul class="question column">
+    <li>Have a question? <a href="#contact"> Click here</a> to contact us!</li>
+</ul>
+
+<!------------------------------------------------ Content main ------------------------------------------------------->
+
+<main class="inner-container column">
+    <div class="triangle-top-right"></div>
+    <div class="work"></div>   <!--  div for big font "work"  -->
+
+    <!------------------------------------------------ Content work --------------------------------------------------->
+    <div id="work" class="background">
+        <section class="flex-container row">
```

Comment:
De section tag gebruik je om content te groeperen, in dit geval zou je dus de section tag plaatsen vanaf regel 52 tot 87.